code-challenges.test.js

```diff
@@ -65,5 +121,45 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe('vowels', () => {
+    it("takes in a string and logs the index of the first vowel")
+    const vowelTester1 = "learn"
+    const vowelTester2 = "academy"
+    const vowelTester3 = "challenges"
+    expect(vowelTester1).toEqual(vowelTester1)
+    expect(vowelTester2).toEqual(vowelTester2)
+    expect(vowelTester3).toEqual(vowelTester3)
+
+})
```

Comment:
It looks like your test is never invoking the function and comparing to an expected result.  Make sure to think about the intent of the test.